tests/test_database.py

```diff
@@ -135,6 +135,24 @@ def test_get_solution_ids(example_database:Database, caplog):
         assert "Solutions file not found" in caplog.text
     assert len(solution_ids) == 0
 
+def test_load_volume(example_database:Database):
+    subject = "example_subject"
+    volume_id = "example_volume"
+    volume = example_database.load_volume(subject, volume_id)
+    assert(volume.id == "example_volume")
+    assert(volume.name == "EXAMPLE_VOLUME")
+
+def test_write_volume(example_database:Database):
+    subject = "example_subject"
+    volume_id = "example_volume_2"
+    volume_name = "EXAMPLE_VOLUME_2"
+    volume_data_path = Path(__file__).parent/"resources/test_db_files/example_volume_2.nii"
+    example_database.write_volume(subject, volume_id, volume_name, volume_data_path)
+    assert(example_database.get_volume_ids("example_subject") == ["example_volume", "example_volume_2"])
+
+    volume_filepath = example_database.get_volume_filepath("example_subject", "example_volume_2")
+    assert(volume_filepath.name == "example_volume_2.json")
+    assert((volume_filepath.parent/"example_volume_2.nii").exists())
```

Comment:
I think you can do this test without introducing the `example_volume_2.nii` resource. The test doesn't need a real nii file, it can just be an empty file. Add a parameter `tmp_path:Path` to `test_write_volume`, similar to how it's done in the `example_database` fixture. This `tmp_path` parameter is a special name for pytest, it will be a temporary directory that you can use just for this test. You can use [`Path.touch`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.touch) to just make an empty file that is then passed as the volume file, and you can give it an nii extension or whatever.